pkg/ocm/config.go

```diff
@@ -50,9 +93,17 @@ func GetEnv() (string, error) {
 		urlAliases = fedramp.URLAliases
 	}
 
-	for env, api := range urlAliases {
-		if api == strings.TrimSuffix(cfg.URL, "/") {
-			return env, nil
+	// Check for OCM environments (including regionalized URLs)
+	if strings.HasSuffix(strings.TrimSuffix(cfg.URL, "/"), "openshift.com") {
```

Comment:
This does not work with localhost urls

@tirthct

Reply:
Hmm, seems like a bug. Created [OCM-6926](https://issues.redhat.com/browse/OCM-6926) and starting to fix the same.